walkeeper/src/replication.rs

```diff
@@ -221,6 +248,28 @@ impl ReplicationConn {
         };
         info!("Start replication from {:?} till {:?}", start_pos, stop_pos);
 
+        // Don't spam pageserver with callmemaybe queries
+        // when connection to compute node is already established.
```

Comment:
Comment is a bit misleading to me, we suspend callmemaybe polling when pageserver connects to us (safekeeper) and the polling will be continued if/when replication connection was destroyed